Packages/manifest.json

```diff
@@ -21,15 +21,18 @@
     "com.unity.ide.visualstudio": "2.0.17",
     "com.unity.ide.vscode": "1.2.5",
     "com.unity.inputsystem": "1.4.4",
+    "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main",
```

Comment:
Revert changes so in sorted order again.

Reply:
I'm having issues doing this in Git; let's handle it later.